torchaudio/__init__.py

```diff
@@ -34,6 +35,30 @@
     pass
 
 
+@_mod_utils.deprecated(
+    "Use `torchaudio.sox_effects.init_sox_effects`, or simply remove the call. "
+    "It is automatically handled."
```

Comment:
I agree that recommending to remove the function call should be the first thing, instead of second option. but I argue `this function doesn't do anything.` is quite accurate and `shutdown_sox_effects` allows user code to release the resources early.